konduit-serving-data/konduit-serving-image/src/main/java/ai/konduit/serving/data/image/step/point/perspective/convert/PerspectiveTransformStep.java

```diff
@@ -46,49 +47,61 @@
     /**
      * If null: just find any Points, Bounding Boxes and Images
      */
+    @Schema(description = "A list of names of the input placeholders.")
     private List<String> inputNames;
 
+    @Schema(description = "A list of names of the output arrays - i.e., what should be predicted.")
```

Comment:
`A list of names of the output field. If not set, the output has the same name as the input field`